taqo_client/lib/main.dart

```diff
@@ -99,9 +99,35 @@ void main() async {
   // This also solves the issue with not having Pending (launch) Intents on Linux
   final activeNotification = await taqo_alarm.checkActiveNotification();
   final authState = await GoogleAuth().isAuthenticated;
+
+  // Add licenses manually for Taqo and Alerter. This help to add the licenses
+  // those are not automatically fetched and added by showLicensePage method.
+  loadManualLicenses();
```

Comment:
async methods should be awaited (or explicitly unawaited), see https://dart.dev/tools/linter-rules#unawaited_futures

I think await is more suitable here.

Reply:
Thank you, I will do this.

Reply:
Incorporated the change.